modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -377,6 +380,7 @@ public String toParamName(String name) {
 
     @Override
     public String toModelName(String name) {
+        name = super.toModelName(name);
```

Comment:
Would there be an issue if we don't call `super.toModelName`, which is defined as follows?
```
    public String toModelName(String name) {
        return initialCaps(name);
    }
```